frontend/src/components/SearchLozenge/SearchLozenge.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './SearchLozenge.css';
+
+const SearchLozenge = (props) => {
+  let { searchTerm } = props;
+  if (!searchTerm) {
+    searchTerm = '';
+  }
+  const lozenges = searchTerm.split(' ');
+  return (
+    <div className="searchLozenge container">
+      <div className="row">
+        <div className="lozengeBubbles">
+          {lozenges.map(lozenge => (
+            <div className="lozenges">
```

Comment:
Maybe just singular "lozenge" as the class name, since there's only one per div?